### PR TITLE
LC1004

### PR DESCRIPTION
**Pull request by janicelmm**
Solution to LC1004

### CODE REVIEW

File: src/Leetcode1004MaxConsecutiveOnesIII.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+public class Leetcode1004MaxConsecutiveOnesIII {
+    public static int longestOnes(int[] nums, int k) 
+    {
+        int j = 0;
+        int i = 0;
+
+        while (i < nums.length)
```
Comment (Ewlai) [inline src/Leetcode1004MaxConsecutiveOnesIII.java]: shouldn't you use a for loop instead of a while loop in this situation?

**Review by Ewlai — CHANGES_REQUESTED** [review]
see comment

File: src/Leetcode1004MaxConsecutiveOnesIII.java (hunk 1 above)
Comment (janicelmm) [reply src/Leetcode1004MaxConsecutiveOnesIII.java]: fixed

**Review by janicelmm — COMMENTED** [review]
fixed

**Review by Ewlai — APPROVED** [review]
LGTM

**Pull request merged**